src/webhook.rs

```diff
@@ -32,17 +35,31 @@ impl Webhook {
         }
     }
 
-    pub async fn execute(
+    pub async fn alert(
         &self,
-        location: &Location,
-        content: impl Display,
+        errors: &[(&Location, HandleLocationError)],
     ) -> Result<(), WebhookExecuteError> {
+        let mut embed = EmbedBuilder::new()
+            .color(0x9E2C2C)
+            .description("Some errors occurred.\nWe notify again when everything is green again.");
```

Comment:
The description does not sound right and using colors to describe a status feels wrong in a text.

```suggestion
            .description("Some errors occurred.\nAs soon as all requests are successful again you will be notified.");
```